src/js/media-error.js

```diff
@@ -100,4 +119,95 @@ for (let errNum = 0; errNum < MediaError.errorTypes.length; errNum++) {
   MediaError.prototype[MediaError.errorTypes[errNum]] = errNum;
 }
 
+// jsdocs for instance/static members added above
```

Comment:
Can we note that this is why it will look like the following is doubled? i.e., we have both `MediaError#MEDIA_ERROR_CUSTOM` (instance) and `MediaError.MEDIA_ERROR_CUSTOM` (static).